packages/wallet/src/KeyManagement/InMemoryKeyAgent.ts

```diff
@@ -23,14 +23,11 @@ export interface InMemoryKeyAgentProps {
 export interface FromBip39MnemonicWordsProps {
   networkId: Cardano.NetworkId;
   mnemonicWords: string[];
+  mnemonic2ndFactorPassword?: Uint8Array;
```

Comment:
```suggestion
  mnemonic2ndFactorPassphrase?: Uint8Array;
```

Reply:
This is the terminology used in the spec https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki. I'll make a ticket to replace the other "password" with "passphrase" for consistency.